cypress/e2e/trafficlight/trafficlight.spec.ts

```diff
@@ -25,6 +25,10 @@ type JSONValue =
     | { [x: string]: JSONValue }
     | Array<JSONValue>;
 
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
Per : https://github.com/matrix-org/trafficlight-adapter-element-web/pull/4#issuecomment-1270048846

I'm not sure if we want to do this; last time it came up it was a real issue in element web that should be caught.

Reply:
Though when i was doing it myself in the other PR; i was thinking along the lines that it doesn't matter so long as the end user experience is OK within the context of the test... so ... not sure.

Odd that this has popped up twice in two weeks though.

Reply:
Yeah, let's actually put it in then.